src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -543,13 +548,27 @@ uct_cuda_copy_md_open(uct_component_t *component, const char *md_name,
         return UCS_ERR_NO_MEMORY;
     }
 
-    md->super.ops              = &md_ops;
-    md->super.component        = &uct_cuda_copy_component;
-    md->config.alloc_whole_reg = config->alloc_whole_reg;
-    md->config.max_reg_ratio   = config->max_reg_ratio;
-    *md_p                      = (uct_md_h)md;
+    md->super.ops               = &md_ops;
+    md->super.component         = &uct_cuda_copy_component;
+    md->config.alloc_whole_reg  = config->alloc_whole_reg;
+    md->config.max_reg_ratio    = config->max_reg_ratio;
+    md->config.dmabuf_supported = uct_cuda_copy_md_is_dmabuf_supported();
```

Comment:
don't override dmabuf_supported var
see how ROCm is doing it..

Reply:
why is this better?

Reply:
easier to track the logic when md->config.dmabuf_supported is set to the right value once